src/createEventEmitterProxy.test.ts

```diff
@@ -160,4 +179,63 @@ describe('createEventEmitterProxy', () => {
 
     expect(proxy.bar()).toStrictEqual([true, 42]);
   });
+
+  it('only migrates events that were added via the proxy, not all the events', () => {
+    const original = new EventEmitter();
+    const next = new EventEmitter();
+
+    let moveCount = 0;
+    original.on('shouldNotMove', () => (moveCount += 1));
+
+    const proxy = createEventEmitterProxy(original);
+    expect(proxy.eventNames()[0]).toBe('shouldNotMove');
+    proxy.on('shouldMove', () => (moveCount += 1));
```

Comment:
thoughts on using a new variable to keep track instead of reusing `moveCount`?

Reply:
should make things a bit more clear